resources/views/account/partials/register-account.blade.php

```diff
@@ -37,6 +37,29 @@
                 v-model="variables.password"
                 required
             />
+            <toggler>
```

Comment:
Is there any specific reason we're using the toggler component here?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details
or the checkbox trick might work just as well without adding to the JS parsing and execution time.

Though i do not know the consensus between toggler vs checkbox vs details

Reply:
Yes, the specific reason is that I want to put `required` on the input, but if you just hide it with css then it still prevents submitting the form.